scripts/release/validatedependenciesversions.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { glob } = require( 'glob' );
+const { promises: { readFile } } = require( 'fs' );
+const chalk = require( 'chalk' );
+const semver = require( 'semver' );
+const { normalizeTrim } = require( 'upath' );
+const isCKEditor5Package = require( './isckeditor5package' );
+
+/**
+ * Validates if the versions of packages and their dependencies in specified directory match the provided version.
+ *
+ * @param {Object} options
+ * @param {String} releaseDirectory Path to directory with packages to validate.
+ * @param {String} options.version Version that all packages and their dependencies need to match.
+ */
+module.exports = async function validateDependenciesVersions( { releaseDirectory, version } ) {
```

Comment:
In other scripts we use the `packagesDirectory` name for the place where all packages to process are located, or `packagesDirectory` + `outputDirectory` if both locations are needed. What do you think about changing the name `releaseDirectory` → `packagesDirectory`?

Reply:
Renamed :)